app/controllers/application_controller.rb

```diff
@@ -213,6 +236,14 @@ def after_sign_in_path_for(resource)
     end
   end
 
+#destroy guest user session if a user signs out, then redirect to root path
+  def after_sign_out_path_for(resource)
+    if session[:guest_user_id]
+      session[:guest_user_id] = nil
```

Comment:
Does this mean we don't transfer the deeds to the permanent guest user if they sign out?